tools/integration-test-baseline.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+require_once __DIR__ . '/../vendor/autoload.php';
+
+copyTests(__DIR__ . '/../tests');
+
+function copyTests(string $directory): void
+{
+    $finder = new \Symfony\Component\Finder\Finder();
+    $finder
```

Comment:
I think that can be done easier:
```php
$finder
  ->directories()
  ->in($directory)
  ->name('input')
```

This way you don't need a recursion, but get all input dirs inside the tests folder at once.
  

https://symfony.com/doc/current/components/finder.html#file-name